app/features/analysis/calculate/OneRMCalculateActions.js

```diff
@@ -78,8 +78,8 @@ export const calcE1RM = () => (dispatch, getState) => {
     const results = OneRMCalculator.calculate1RM(exercise, tagsToInclude, tagsToExclude, daysRange, velocity, metric, allSets);
 
     // analytics
-    logCalculate1RMAnalytics(
-        state,
+
+    const calculateE1RMParams = analyticsParams(
```

Comment:
- [ ] analyticsParams

Reply:
I made it oneRMAnalyticsParams since there is already a function called analyticsParams